Gemfile

```diff
@@ -8,3 +8,5 @@ gemspec
 gem "rake", "~> 13.0"
 
 gem "zeitwerk", "~> 2.5"
+
+gem "dotenv", "~> 2.7"
```

Comment:
I did not catch it on the other PR, but when working on gems, dependencies should be defined in the .gemspec file.

Reply:
And it looks like i did the mistake of not adding the `gemfile.lock` file to the `gitignore`. I will fix that one.